thorvision/src/server_status_indicator.cc

```diff
@@ -23,6 +23,8 @@ ServerStatusIndicator::ServerStatusIndicator(QWidget *parent)
 
     _thread = std::jthread([this, status_text]() {
         auto const timeout = 500ms;
+        int retry_count = 0;
+        int const max_retries = 90;

```

Comment:
```suggestion
        int const max_retries = 10;
```
Reduce max_retries to 10